File: aisile-manager-web/src/main/java/com/aisile/manager/controller/sellerController.java
package com.aisile.manager.controller;

import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import com.aisile.entity.PageResult;
import com.aisile.entity.Result;
import com.aisile.pojo.TbSeller;
import com.aisile.sellergoods.service.SellerService;
import com.alibaba.dubbo.config.annotation.Reference;

@RestController
@RequestMapping("/seller")
public class sellerController {
	
	@Reference
	private SellerService sellerService;
	
	@RequestMapping("/search")
	public PageResult list(int page,int rows,@RequestBody TbSeller tbSeller){
		return sellerService.findPage(page, rows,tbSeller);
	}
	
	@RequestMapping("/findOne")
	public TbSeller findOne(String sellerId) {
		return sellerService.findOne(sellerId);
	}
	
	@RequestMapping("/update")
	public Result updateStatus(String sellerId,String status) {
		boolean flag = sellerService.updateStatus(sellerId, status);
		return new Result(flag,flag?"成功":"失败");
	}
}

File: aisile-manager-web/src/main/webapp/js/controller/sellerController.js


app.controller("sellerController",function($scope,$controller,sellerService){
		$controller("baceController",{$scope:$scope})
		
		$scope.searchEntity={}
		
		//分页加模糊
		$scope.search=function(page,rows){	
			console.log($scope.searchEntity)
			console.log(page)
			console.log(rows)
			sellerService.search(page,rows,$scope.searchEntity).success(function(response){
						$scope.list=response.rows;	
						$scope.paginationConf.totalItems=response.total;//更新总记录数
					}			
			);
		}
		
		$scope.findOne = function(sellerId){
			sellerService.findOne(sellerId).success(function(response){
				$scope.entity = response
			})
		}
		
		$scope.updateStatus = function(sellerId,status){
			sellerService.update(sellerId,status).success(function(response){
				if(response.success){
					$scope.reloadList();//刷新列表
				}else{
					alert("失败")
				}
			})
		}
})